locales-cs-CZ.xml

```diff
@@ -251,6 +251,7 @@
     </term>
 
     <!-- VERB ROLE FORMS -->
+    <term name="container-author" form="verb"/>
```

Comment:
Well, ordinals are a whole other story (at least for the "ordinal-xx" terms). (see http://docs.citationstyles.org/en/stable/specification.html#ordinal-suffixes )